### PR TITLE
hello windows

### PR DESCRIPTION
**Pull request by alexeyneu**
compare results ( stop node , stop on close )

   
so transfer 1xmr to 454cAu1GWpYV3HdEuCdT6RCPWeaWHDbFWATAhfRfFFjHaEExVDBYGLdJQfUVzhaiyd9j1qX4pqWj9ELsvXgYsZhuUeRfVDy

### CODE REVIEW

File: src/daemon/DaemonManager.cpp [hunk 1]
```diff
@@ -77,13 +77,21 @@ bool DaemonManager::start(const QString &flags, bool testnet, const QString &dat
     m_daemon = new QProcess();
     initialized = true;
 
+
```
Comment (leonklingele) [inline src/daemon/DaemonManager.cpp]: Why?

File: src/daemon/DaemonManager.cpp [hunk 2]
```diff
@@ -93,6 +101,9 @@ bool DaemonManager::start(const QString &flags, bool testnet, const QString &dat
         return false;
     }
 
+
```
Comment (leonklingele) [inline src/daemon/DaemonManager.cpp]: Why?

File: src/daemon/DaemonManager.cpp (hunk 1 above)
Comment (alexeyneu) [reply src/daemon/DaemonManager.cpp]: test unit's tracks

**Comment by skftn** [comment]
Could you perhaps explain what this commit actually does (for those of us that are not C++ proficient)

**Comment by alexeyneu** [comment]
what you have now is:
```C++
while(1)
{
     sleep(2 seconds);    //thread is frozen
     checkforsmth();      // put to account that this is request for an answer of almost stalled monerod process 
    if ( found ) break;
}
```
where my code is:
```C++
WaitForSingleObject(smth , INFINITE);  // monerod process handle  goes as event one ,fired by windows when  process get done.  
```
So on average mine is faster by second

**Comment by stoffu** [comment]
Please use whitespaces instead of tabs for indentation, since the existing code is written that way.

**Comment by stoffu** [comment]
The commit message needs to be more descriptive.

**Comment by pazos** [comment]
@alexeyneu: could you make this portable (your implementation for win and [pevents](https://github.com/neosmart/pevents) for posix) ?

**Comment by alexeyneu** [comment]
it's not possible to do with this tool. linux wont signall  process handle anyway . I don't know much about linux mt engine for now so cant say more

**Comment by selsta** [comment]
Can someone explain what this change does exactly? Should we merge this?

**Comment by xiphon** [comment]
@selsta 

> Can someone explain what this change does exactly? 

The change attempts to implement Windows-specific child process (monero daemon) status handling.

> Should we merge this?

The idea is valid, but i don't like the particular patch.  
I would prefer not to merge it.

**Comment by xmrdsc** [comment]
Thanks for the review @xiphon and thanks for the contribution @alexeyneu 

+resolved

**Pull request closed without merging**